lib/string.gd

```diff
@@ -860,6 +860,73 @@ BindGlobal("BHINT", MakeImmutable("\>\<"));
 
 DeclareGlobalFunction("StringOfMemoryAmount");
 
+#############################################################################
+##
+##  <#GAPDoc Label="StringFormatted">
+##  <ManSection>
+##  <Func Name="StringFormatted" Arg='string, data...'/>
+##  <Func Name="PrintFormatted" Arg='string, data...'/>
+##  <Func Name="PrintToFormatted" Arg='stream, string, data...'/>
+##
+##  <Description>
+##  These functions format an input string.
+##  <Ref Func="StringFormatted"/> returns the formatted string,
+##  <Ref Func="PrintFormatted"/> prints the formatted string and
+##  <Ref Func="PrintToFormatted"/> appends the formatted string to <A>stream</A>,
+##  which can be either an output stream or a filename.
+##  <P/>
+##  <A>data</A> is a list of values to printed, using the formatting rules below:
+##  <P/>
+##  <A>string</A> is treated as a normal string, except for occurrences
+##  of <C>{</C> and <C>}</C>, which follow special rules, as follows:
+##
+##  The contents of <C>{ }</C> form a small language. The format is <C>{id!format}</C>,
+##  where both <C>id</C> and <C>format</C> are optional. If the <C>!</C> is ommitted, the bracket
+##  is treated as <C>{id}</C> with no <C>format</C>.
+##  <P/>
+##  <C>id</C> is interpreted as follows:
+##  <List>
+##  <Mark>An integer <C>i</C></Mark> <Item>
+##    Take the <C>i</C>th element of <A>data</A>.
+##  </Item>
+##  <Mark>No variable</Mark> <Item>
```

Comment:
What does "No variable" mean here? Perhaps "Not given" or, better, "No id given" ?